kurtosis-devnet/justfile

```diff
@@ -82,6 +82,13 @@ devnet TEMPLATE_FILE DATA_FILE="" NAME="" PACKAGE=KURTOSIS_PACKAGE:
         fi
     fi
     export ENCL_NAME="$DEVNET_NAME"-devnet
+
+    if [[ -z "$SKIP_ENCL_CLEAN" ]]; then
+        # clear out any existing enclaves with the same name.
+        # this can fail and it doesn't matter.
+        kurtosis enclave rm "$ENCL_NAME" --force 2&>/dev/null || true
```

Comment:
I'm not sure it'll help all that much.

In scenarios where re-deploying the enclave in-place doesn't work, there's a decent chance that `kurtosis enclave rm` won't work either: typically it's not the enclave itself that's the problem, but the surrounding kurtosis/docker environment

Reply:
Interesting. I'm under the impression that this command also nukes the containers. Is that incorrect?